org.gnome.Keysign.yml

```diff
@@ -0,0 +1,311 @@
+    build-commands:
+      - pip2 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} pytz
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/10/76/52efda4ef98e7544321fd8d5d512e11739c1df18b0649551aeccfb1c8376/pytz-2018.4.tar.gz
+        sha256: c06425302f2cf668f1bba7a0a03f3c1d34d4ebeef2c72003da308b3947c7f749
+
+  - name: pygobject
+    build-options:
+      env:
+        PYTHON: python2
+    ensure-writable:
+      - easy-install.pth
+      - setuptools.pth
+    sources:
+      - type: archive
+        url: http://ftp.acc.umu.se/pub/GNOME/sources/pygobject/3.28/pygobject-3.28.2.tar.xz
+        sha256: ac443afd14fcb9ff5744b65d6e2b380e70510278404fb8684a9b9fb089e6f2ca
+
+  - name: swig
+    config-opts:
+      - "--without-boost"
+      - "--without-pcre"
+    sources:
+      - type: archive
+        url: http://prdownloads.sourceforge.net/swig/swig-3.0.12.tar.gz
+        sha256: 7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d
+
+  - name: dbus-python
+    ensure-writable:
+      - easy-install.pth
+      - setuptools.pth
+    sources:
+      - type: archive
+        url: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.2.4.tar.gz
+        sha256: e2f1d6871f74fba23652e51d10873e54f71adab0525833c19bad9e99b1b2f9cc
+
+  - name: avahi
+    # hrm, I don't want to install this, but we need the python module
+    disabled: false
+    rm-configure: true
+    config-opts:
+      - "--enable-python"
+      - "--disable-static"
+      - "--disable-mono"
+      - "--disable-monodoc"
+      - "--disable-qt3"
+      - "--disable-qt4"
+      - "--disable-gtk"
+      - "--disable-gtk3"
+      - "--enable-core-docs"
+      - "--with-distro=none"
+      - "--with-systemdsystemunitdir=no"
+      - "--disable-libdaemon"
+      - "--disable-pygtk"
+      - "--disable-gdbm"
+    ensure-writable:
+      - easy-install.pth
+      - setuptools.pth
+    sources:
+      - type: archive
+        url: http://avahi.org/download/avahi-0.6.31.tar.gz
+        sha256: 8372719b24e2dd75de6f59bb1315e600db4fd092805bd1201ed0cb651a2dab48
+      - type: script
+        commands:
+        - autoreconf -fi
+        dest-filename: autogen.sh
+
+  - name: gstreamer
+    sources:
+      - type: archive
+        url: https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.12.3.tar.xz
+        sha256: d388f492440897f02b01eebb033ca2d41078a3d85c0eddc030cdea5a337a216e
+
+  - name: gst-plugins-base
+    sources:
+      - type: archive
+        url: https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.12.3.tar.xz
+        sha256: d3d37b8489d37fa0018973d850bd2067b98af335fef2fa543ee7d40359e3cea5
+
+  - name: gst-plugins-good
+    sources:
+      - type: archive
+        url: https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.12.3.tar.xz
+        sha256: 13e7f479296891fef5a686438f20ba7d534680becf2269ecc5ee24aa83b45f03
+
+  - name: gst-plugins-ugly
```

Comment:
I feel like a keysign tool doesn't need every codec gstreamer offers. Limiting it to the few plugins actually needed would be nice.